### ISSUE TITLE
Fine-tuning with segtrain does not start "WARNING Setting baseline location to baseline"

### ISSUE THREAD

**Issue by DenisaFB** [mittagessen/kraken#764]
I am trying to fine-tune the Kraken base model blla.mlmodel specifically for baselines and line masks. However, I am unsure whether I am using the correct command, as the fine-tuning process does not appear to start. My goal is to specialize the model so that it can recognize the class InterlinearLine as well.

Also I should mention the images are pretty large, should I indicate a minimum image size? If we have double pages images of manuscripts, the size is around "4000x4000". Lines of the type InterlinearLine can be very small compared to normal lines of text.

Kraken version: 6.0.3
Total number of GT images : 36
Split train/valid : train 32, valid : 4

I run the following segtrain command without prepared data split : 

`!ketos -d cuda:0 --workers 3 segtrain --logger tensorboard  -i /content/pretrained_models/blla.mlmodel -f page --resize new /content/data/*.xml -o /content/checkpoints/ft_leibniz_v1`.

And this command with the data split: 

`!ketos -d cuda:0 --workers 3 segtrain --logger tensorboard -i /content/pretrained_models/blla.mlmodel -f page -t /content/train.txt -e /content/val.txt --suppress-regions -q early --min-epochs 50 --resize new -bl -o /content/checkpoints/ft_leibniz_v1`

In both cases the fine-tuning stops at `[02/24/26 15:21:57] WARNING  Setting baseline location to baseline`

Terminal : 

```
2026-02-24 15:21:49.569860: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:467] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
E0000 00:00:1771946509.592038   32574 cuda_dnn.cc:8579] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
E0000 00:00:1771946509.599539   32574 cuda_blas.cc:1407] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
W0000 00:00:1771946509.619016   32574 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.
W0000 00:00:1771946509.619044   32574 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.
W0000 00:00:1771946509.619053   32574 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.
W0000 00:00:1771946509.619059   32574 computation_placer.cc:177] computation placer already registered. Please check linkage and avoid linking the same target more than once.
2026-02-24 15:21:49.623908: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
Training line types:
  DefaultLine	2	5083
  InterlinearLine	3	870
Training region types:
  MainZone	4	70
  MarginTextZone	5	195
  DigitizationArtefactZone	6	36
  NumberingZone	7	43
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
HPU available: False, using: 0 HPUs
`Trainer(val_check_interval=1.0)` was configured so validation will run at the end of the training epoch..
WARNING:py.warnings:/usr/local/lib/python3.12/dist-packages/lightning/pytorch/utilities/parsing.py:44: Attribute 'progress_callback' removed from hparams because it cannot be pickled. You can suppress this warning by setting `self.save_hyperparameters(ignore=['progress_callback'])`.

WARNING:py.warnings:/usr/local/lib/python3.12/dist-packages/lightning/pytorch/utilities/parsing.py:44: Attribute 'message' removed from hparams because it cannot be pickled. You can suppress this warning by setting `self.save_hyperparameters(ignore=['message'])`.

[02/24/26 15:21:57] WARNING  Setting baseline location to baseline 
```

Here is the hparams.yaml file : 

```
bounding_regions: !!python/tuple []
evaluation_data:
- /content/data/LH_1_3_7_C_0001.xml
- /content/data/LH_1_4_1_0001r.xml
- /content/data/LH_1_20_0111v-0110r.xml
- /content/data/LH_1_2_1_0025r-0024v.xml
force_binarization: false
format_type: page
hyper_params:
  augment: false
  completed_epochs: 0
  cos_min_lr: 2.0e-05
  cos_t_max: 50
  epochs: 50
  freq: 1.0
  gamma: 0.1
  lag: 10
  line_width: 8
  lrate: 0.0002
  min_delta: null
  min_epochs: 50
  momentum: 0.9
  optimizer: Adam
  padding: !!python/tuple
  - 0
  - 0
  quit: early
  rop_factor: 0.1
  rop_patience: 5
  schedule: constant
  step_size: 10
  warmup: 0.0
  weight_decay: 1.0e-05
load_hyper_parameters: false
merge_all_baselines: null
merge_all_regions: null
merge_baselines: null
merge_regions: null
message: !!python/name:kraken.lib.train.%3Clambda%3E ''
model: /content/pretrained_models/blla.mlmodel
num_workers: 3
output: /content/checkpoints/ft_leibniz_v1
partition: 1
progress_callback: !!python/name:kraken.lib.train.%3Clambda%3E ''
resize: new
spec: '[1,1800,0,3 Cr7,7,64,2,2 Gn32 Cr3,3,128,2,2 Gn32 Cr3,3,128 Gn32 Cr3,3,256 Gn32
  Cr3,3,256 Gn32 Lbx32 Lby32 Cr1,1,32 Gn32 Lby32 Lbx32]'
suppress_baselines: false
suppress_regions: true
topline: false
training_data:
- /content/data/LH_1_3_1_0003r-0002v.xml
- /content/data/LH_1_3_4_0011r-0010v.xml
- /content/data/LH_1_2_1_0004v.xml
- /content/data/LH_1_6_5_0002v-0001r.xml
- /content/data/LH_2_3_1_0054r-0053v.xml
- /content/data/LH_1_4_3_0004r-0003v.xml
- /content/data/LH_1_7_6_0043r-0042v.xml
- /content/data/LH_1_6_2_0002v-0001r.xml
- /content/data/LH_1_4_7_0015v-0014r.xml
- /content/data/LH_1_6_4_0002r-0001v.xml
- /content/data/LH_2_3_1_0051r-0050v.xml
- /content/data/LH_1_3_1_0008v-0007r.xml
- /content/data/LH_1_3_4_0008r-0007v.xml
- /content/data/LH_1_6_6_0002v-0001r.xml
- /content/data/LH_1_2_1_0027r.xml
- /content/data/LH_1_4_4_0001v.xml
- /content/data/LH_1_3_7_D_0002v-0001r.xml
- /content/data/LH_4_6_12a_0001v.xml
- /content/data/LH_1_2_1_0005r.xml
- /content/data/LH_1_6_11_0002r-0001v.xml
- /content/data/LH_1_20_0065v-0064r.xml
- /content/data/LH_4_1_4d_0002r-0001v.xml
- /content/data/LH_1_3_5_0024v-0023r.xml
- /content/data/LH_1_4_3_0002v-0001r.xml
- /content/data/LH_1_20_0164v-0163r.xml
- /content/data/LH_2_3_1_0058r-0057v.xml
- /content/data/LH_1_7_5_0065r-0064v.xml
- /content/data/LH_1_4_7_0012v-0011r.xml
- /content/data/LH_1_20_0287v-0284r.xml
- /content/data/LH_1_4_8_0016r-0015v.xml
- /content/data/LH_1_2_1_0004r.xml
- /content/data/LH_1_3_7_A_0004v-0003r.xml
valid_baselines: !!python/tuple []
valid_regions: !!python/tuple []
```
